init/setup-rw.sh

```diff
@@ -72,6 +72,13 @@ if [ -L /home ]; then
     mkdir /home
 fi
 
+# Old Qubes versions had symlink /usr/local -> /rw/usrlocal; now we use mount --bind
+if [ -L /usr/local ]; then
+    rm /usr/local
+    mkdir /usr/local
+fi
+
+
```

Comment:
This should be at packaging level (rpm spec, debian postinst, archilinux install scripts), not at every VM start.

Reply:
Ok. I went with how the `/home` upgrade seemingly was handled. I'm not sure how you mean. Should I remove this commit? Thanks

Reply:
Yes, please.

And when you pointed /home handling there I think I should move it to package scripts too. Or remove completely (we've changed /home to a bind mount in 2012...).